pom.xml

```diff
@@ -133,6 +136,11 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```

Comment:
Why we need this dependency ? beware typo : ${junit-jupiter-engine.version}} (double brackets)

Reply:
We don't, it is something that I try in order to be able to generate idea:module with maven, I will remove it.